### DISCUSSION TITLE
Print using random colors

### DISCUSSION THREAD

**Discussion by Justaus3r** [Textualize/rich#2097] [Ideas]
i have a feature request which i think haven't been requested before or so i found after searching through issues and discussions.
have a method like ``.print_random`` that prints the text using a random color from supported color list.

**Comment by willmcgugan** [comment]
Don't think thats worthy of going in the core lib, but you could implement that quite easily.

**Reply by Justaus3r** [reply]
i see.yeah it is pretty easy,just wanted to know if you were interested in implementing it in main lib.anyways thanks for the reply
